### ISSUE TITLE
Root NS support not in published package

### ISSUE THREAD

**Issue by rtokarek-fastly** [octodns/octodns-ns1#32]
It looks like the most recent release was February last year which is before root NS record support was toggled and merged into the main branch. 

Is there any chance we could get a new release with that change? (It looks like it's prepared in the changelog, along with a number of other changes which might be causing the delay). We can work around this, but we really do want the octodns providers (of which ns1 is one) we're using to support making changes to root NS records. 

https://github.com/octodns/octodns-ns1/pull/11

Thanks! If there's anything we can do to help the process, lemme know.

**Comment by ross** [comment]
Don't see anything that's preventing a release from being cut so I'll queue that up.

Fwiw, I recommend using pinned git versions for grabbing things when there's not a release with exactly what you want. We pretty much exclusively did so for octoDNS at GitHub to avoid having to wait on releases (even though I was the one doing them.)
